source/write/replace.txt

```diff
@@ -0,0 +1,202 @@
+.. _kotlin-sync-write-replace:
+
+=================
+Replace Documents
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: modify, change, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to perform a **replace
+operation** on a document in a MongoDB collection. A replace operation
+removes all fields and values from a specified document except the
+``_id`` field, and adds new fields and values that you specify. This
+operations differs from an update operation, which changes only
+specified fields in one or more documents.
+
+.. tip:: Learn About Update Operations
```

Comment:
S: Make this part of the text rather than a callout, I'm unsure if making this is a callout is necessary